docs/contracts/interfaces.rst

```diff
@@ -37,10 +37,32 @@ they can be overridden. This does not automatically mean that an overriding func
 can be overridden again - this is only possible if the overriding
 function is marked ``virtual``.
 
+Interfaces can inherit from other interfaces. This, in general, has the same rules as
+for normal contracts, except that an interface does not need to override a function
```

Comment:
This exception has been removed, has it?

Reply:
Yes, yes it has. Gah, documentation is hard :stuck_out_tongue:.